nicsdru_nidirect_theme.theme

```diff
@@ -338,6 +338,21 @@ function nicsdru_nidirect_theme_preprocess_page_title(&$variables) {
 
   switch ($route->getRouteName()) {
 
+    // Preprocess search page title.
+    case "view.search.search_page":
+      // Modify title if a search query is present.
+      $query = $request->get('query');
+      if (isset($query)) {
```

Comment:
A nitpick, but I tend to find contrib and core prefer `empty()` over `isset()` because the latter can sometimes return TRUE even if the underlying value is actually FALSE simply because the variable has been assigned/set rather than testing on whether it's got a useful value. See http://www.drupal8.ovh/en/tutoriels/228/isset-vs-empty-vs-null

Reply:
Yep empty() does seem more appropriate here. Have fixed that.